_projects/bbc-ma.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Nah, it is a first draft to see what is commonly needed before settling with a formal structure.